### PR TITLE
[strings] Translate separate address nodes as Address instead of Building

### LINKED ISSUE #6954: "Building" is displayed for addr:street + addr:housenumber
A simple fix is to write "Address" instead of "Building".

### PR DESCRIPTION
**Pull request by biodranik**
Fixes #6954

### CODE REVIEW

**Review by vng — APPROVED** [review]

**Comment by d4f5409d** [comment]
Is that this type of string?

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -6385,7 +6385,47 @@
...
+    id = Alamat
+    it = Indirizzo
+    ja = 住所
+    ko = 주소
+    lt = Adresas
+    mr = पत्ता
+    nb = Adresse
+    nl = Adres
+    pl = Adres
+    pt = Endereço
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:6417]:
```suggestion
    pt = Morada
    pt-BR = Endereço
```

**Comment by biodranik** [comment]
> Is that this type of string?

No, it's for stand-alone addresses.

**Comment by biodranik** [comment]
Updated, we already have these translations, not sure if it's ok to reuse them. @vng PTAL

**Review by jimcarst — APPROVED** [review]
ok for nl

**Pull request merged**